kubernetes/beta/hmda-data-browser-api/values.yaml

```diff
@@ -69,5 +69,10 @@ service:
 
 databrowser:
   name: databrowser
+  loglevel: INFO
+  nationwideRouteCSV: https://s3.amazonaws.com/cfpb-hmda-public/prod/data-browser/nationwide_all.csv
```

Comment:
For extensibility it would probably be useful to have the `s3` route broken into pieces that we can reuse if we decide to serve more from there. As well, we already have `cfpb-hmda-public` in our `s3-configmap` if you just want to take it from there.

Reply:
@kgudel Changes made